src/Endpoints/TenantToken.php

```diff
@@ -36,7 +36,7 @@ private function validateTenantTokenArguments($searchRules, ?array $options = []
      * - apiKey: The API key parent of the token. If you leave it empty the client API Key will be used.
      * - expiresAt: A DateTime when the key will expire. Note that if an expiresAt value is included it should be in UTC time.
      */
-    public function generateTenantToken($searchRules, ?array $options = []): string
+    public function generateTenantToken(string $uid, $searchRules, ?array $options = []): string
```

Comment:
Like suggested in this [comment]
(https://github.com/meilisearch/meilisearch-python/pull/484#pullrequestreview-1017277409) it could be good to rename `uid` to `api_key_uid`

Reply:
Thanks @alallema I'll apply this suggestion in a next PR, btw I will do it in the dart as well :)